metadata/README.md

```diff
@@ -0,0 +1,52 @@
+# Fedora CoreOS metadata
+
+Fedora CoreOS artifacts and streams are described by metadata objects, in the form of JSON documents.
+This allows the general audience to consume releases and updates in a machine-friendly way.
```

Comment:
Hmm, should we also lay out in this document which pieces of metadata are meant to be directly used by people, and which are not? E.g. the stream metadata is, but the release index isn't, right?

Reply:
That'd be fine for me. Where would the other two fit?

Reply:
The release metadata also is, and I guess the updates stream too (since we want other Cincinnati servers to read from there) though it's not stable yet.

/cc @bgilbert to sanity check

Reply:
Instead of trying to draw arbitrary lines, I added another commit explaining the usage of each. Does it look clearer that way?